lib/driver.js

```diff
@@ -153,6 +153,16 @@ class AndroidDriver extends BaseDriver {
         await this.checkPackagePresent();
       }
 
+      // check if we have to enable/disable gps before running the application
+      if (this.opts.gpsEnabled) {
```

Comment:
probably want `if (util.hasValue(this.opts.gpsEnabled)) {`, otherwise if someone sets it to `false` this will never pass